packages/waas/src/payloads/signers/index.ts

```diff
@@ -0,0 +1,28 @@
+import { newSECP256K1RandomSigner, newSECP256K1Signer } from "./secp256k1";
+import { newSECP256R1Signer, newSECP256R1RandomSigner } from "./secp256r1";
+
+export type PayloadSigner = {
+  privateKey: string
```

Comment:
lets update the type to have `privateKey?: string` but add `publicKey: string` for k1 curve, you can also get the public key from ethers, or we can just return the address generated. For r1 curve, lets return the public key in hex.

Reply:
At the moment this field is used to persist the key. Not sure how this will look like when we will be using ``IndexedDB``. I guess it's to be determined.